src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleared = sourceString.split(';');
+
+  return cleared.reduce((accum, style) => {
+    const n = style
+      .split(':')
+
+      .map((element) => element.trim());
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. Align the '.map' call with the '.split' call for better readability.